### ISSUE TITLE
Ruby Homebrew installer is now deprecated and should be migrated to Bash

### ISSUE THREAD

**Issue by ehanlon** [chef-cookbooks/homebrew#135]
### Cookbook version
5.0.8

### Chef-client version
15.8.23

### Platform Details
mac_os_x

### Scenario:
`Error executing action `run` on resource 'execute[install homebrew]'`

```bash
STDERR: Warning: The Ruby Homebrew installer is now deprecated and has been rewritten in
           Bash. Please migrate to the following command:
             /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
```

**Comment by bcg62** [comment]
also hitting this issue which is a blocker for users.

**Comment by bbros-dev** [comment]
@ehanlon and @bcg62 you can workaround as follows:

```ruby
url_sh = 'https://raw.githubusercontent.com/Homebrew/install/master/install.sh'
node.default['homebrew']['installer']['url'] = url_sh
include_recipe 'homebrew'
```

HTH?

**Comment by ehanlon** [comment]
Fixed by #140
